src/vs/base/browser/ui/selectBox/selectBoxCustom.ts

```diff
@@ -478,6 +478,22 @@ export class SelectBoxList extends Disposable implements ISelectBoxDelegate, ILi
 		this._currentSelection = this.selected;
 		this._isVisible = true;
 		this.selectElement.setAttribute('aria-expanded', 'true');
+
+		// {{SQL CARBON EDIT}}: Setting aria-selected attribute of dropdown items
+		this.options.forEach((option, index) => {
```

Comment:
Did you get these changes from VS Code or something? If we're making a11y changes to core VS Code stuff we should generally be going through these questions/steps

Is it an issue in VS Code as well? 
- If so then work with them to get a fix and port that into ADS
- If not then figure out why not - are we doing something they don't support? Can we fix this in the SQL code instead of modifying the core code?

Reply:
These changes are not from vscode. The bugs is also present in vscode dropdowns. I have tried the debug dropdown for the repro.

![image](https://user-images.githubusercontent.com/6816294/173702165-38fc7732-b8b5-447d-a047-56f0a993bcb2.png)

Reply:
Please open an issue in the VS Code repo (make sure there isn't one already). 

Then the suggestion I have would be to follow up with them on getting a fix in there (you can start with your proposed fix here) and once that's done port to ADS. 

@alanrenmsft That all sound good to you?

Reply:
yes, that is the right thing to do.